writing-visualising/dashboards.qmd

```diff
@@ -10,9 +10,9 @@ title: "Public dashboards"
 
 ---
 
-The primary route for publishing official statistics is via [Explore education etatistics](https://explore-education-statistics.service.gov.uk/) (EES). In addition to publishing on EES, there may be times where you want to compliment official statistics with a dashboard. This may be to do a deeper piece of analysis for a specific user group, or to make use of functionality that isn’t available within the platform.
+The primary route for publishing official statistics is using the [explore education statistics service](https://explore-education-statistics.service.gov.uk/) (EES). In addition to publishing on EES, there may be times where you want to compliment official statistics with a dashboard. This may be to do a deeper piece of analysis for a specific user group, or to make use of functionality that isn’t available within the platform.
 
-It is important that all dashboards should have a clear use case and reasoning for why they are required. If you are considering developing and publishing a dashboard, please get in touch with the [Statistics Development Team](mailto:statistics.development@education.gov.uk) who can advise and assist with the process, and make use of the [teams channel](https://teams.microsoft.com/l/channel/19%3a35fa27cbced34928a3609e41510057e5%40thread.skype/R-Shiny%2520Statistics%2520Dashboards?groupId=679b2376-8c8c-4062-a1c9-0744ce5ac88f&tenantId=fad277c9-c60a-4da1-b5f3-b3b8b34a82f9) to ask questions and share what you're working on with other teams.
+It is important that all dashboards should have a clear use case and reasoning for why they are required. If you are considering developing and publishing a dashboard, please get in touch with the [explore education statistics platforms team](mailto:explore.statistics@education.gov.uk) who can advise and assist with the process, and make use of the [teams channel](https://teams.microsoft.com/l/channel/19%3a35fa27cbced34928a3609e41510057e5%40thread.skype/R-Shiny%2520Statistics%2520Dashboards?groupId=679b2376-8c8c-4062-a1c9-0744ce5ac88f&tenantId=fad277c9-c60a-4da1-b5f3-b3b8b34a82f9) to ask questions and share what you're working on with other teams.
```

Comment:
ugh nitpicky but I think Teams should have a big T - also should we mention the name of the Teams channel that the link will take them to?

Reply:
Agree on both, fixed in next commit